src/apps/backend/modules/communication/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as EmailService } from './email-service';
+export { default as SMSService } from './sms-service';
```

Comment:
could we add a push notification service as well by default?

Reply:
You are encouraged to raise an issue on this repo. This would be out of scope of this PR.

Reply:
ok